api/tests/test_table.py

```diff
@@ -286,3 +286,87 @@ def test_simple(self):
         self.api_req("get", exp_code=404)
         # create table again so that teardown works
         self.create_table(self.test_structure)
+
+
+class TestMove(APITestCaseWithTable):
+    test_table = "test_table_move"
+    test_schema = "model_draft"  # cannot move from "test" schema
+    target_schema = "scenario"
+
+    def test_move(self):
+        self.api_req(
+            "post",
+            path=f"move/{self.target_schema}/",
+            exp_code=200,
+        )
+
+        # check that target table exists, and source does not exist anymore
+        self.api_req("get", exp_code=404)
+        self.api_req("get", schema=self.target_schema, exp_code=200)
+
+        # move back
+        self.api_req(
+            "post",
+            schema=self.target_schema,
+            path=f"move/{self.test_schema}/",
+            exp_code=200,
+        )
+
+
+class TestMovePublish(APITestCaseWithTable):
```

Comment:
@jh-RLI this is my test case now. hope it makes sense to you

Reply:
<3